### PR TITLE
Further tweak the colors and styles

### PR DESCRIPTION
**Pull request by notriddle**
* Fix the footer pinning behavior, so that it works like the old site

* Use Yellow90 for the text in the alert box

<details><summary>Previous revision of this pull request made more radical changes</summary>

* Fix the footer pinning behavior, so that it works like the old site

* Use Yellow90 for the text in the alert box (as described in the Photon docs)

* Use Gray10 for the site background, reserving White only for pop-ups, floating boxes, and active form fields (as described in the Photon docs)

* Recolor the footer in Photon Ink (this is the part I'm most willing to revert, making the footer the same as the rest of the document)
</details>

### CODE REVIEW

File: static/css/janitor-new.css [hunk 1]
```diff
@@ -4,8 +4,9 @@
   box-sizing: border-box;
 }
 
-html, body {
-  min-height: 100%;
+html {
+  height: 100%;
+  overflow-y: scroll;
```
Comment (jankeromnes) [inline static/css/janitor-new.css]: Interesting, why did you add this rule?

File: static/css/janitor-new.css [hunk 2]
```diff
@@ -16,6 +17,7 @@ body {
   font-size: 14px;
   display: flex;
   flex-direction: column;
+  min-height: 100%;
```
Comment (jankeromnes) [inline static/css/janitor-new.css]: In testing, I was able to fix the "pinned footer" by just using `height: 100%` instead of `min-height: 100%` on just the body, which seems a little simpler, but since I don't know anything about CSS I'll defer the review to experts like @nt1m and @arshdkhn1.

**Comment by jankeromnes** [comment]
Ping @nt1m (please have a look at this when convenient) 😄

File: static/css/janitor-new.css (hunk 2 above)
Comment (nt1m) [reply static/css/janitor-new.css]: I started by using height: 100%, but it seems to squish the header when there is a lot of content in the page.

File: static/css/janitor-new.css [hunk 3]
```diff
@@ -125,12 +129,14 @@ main {
...
   background: #fff;
   color: #0b0b0d;
-  box-shadow: 0 4px 16px rgba(12,12,13,.1);
-  overflow: auto;
   z-index: 999;
   font-size: 1.1em;
+}
+
+.navbar-sticky {
   position: sticky;
```
Comment (arshdkhn1) [inline static/css/janitor-new.css]: `position: sticky` not supported in IE and old browsers?

File: static/css/janitor-new.css [hunk 4]
```diff
@@ -4,8 +4,11 @@
   box-sizing: border-box;
 }
 
-html, body {
-  min-height: 100%;
+html {
+  height: 100%;
+  background-color: #f9f9fa;
+  color: #000;
+  overflow-y: scroll;
```
[the remark this answers is not in the document]
Comment (notriddle) [reply static/css/janitor-new.css]: I want the scroll bar to appear all the time. Otherwise, the centered content jumps around when you switch pages.

[the remark this answers is not in the document]
Comment (arshdkhn1) [reply static/css/janitor-new.css]: @notriddle I also didnt like that jumping but I guess that's fine. Only chrome has that jumping behaviour. there is a margin technique to avoid scrollbar and jumping

**Comment by notriddle** [comment]
Okay. Gray background and ink footer are going away as soon as I'm back at my computer.

File: static/css/janitor-new.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (arshdkhn1) [reply static/css/janitor-new.css]: but that margin technique is not worth the pain.

File: static/css/janitor-new.css (hunk 3 above)
Comment (notriddle) [reply static/css/janitor-new.css]: Actually, it's mostly so that I don't have to override all this stuff on the footer.

**Comment by notriddle** [comment]
Fixed.

File: static/css/janitor-new.css [hunk 5]
```diff
@@ -4,8 +4,11 @@
   box-sizing: border-box;
 }
 
-html, body {
-  min-height: 100%;
+html {
+  height: 100%;
+  background-color: #fff;
+  color: #0b0b0d;
+  overflow-y: scroll;
```
Comment (nt1m) [inline static/css/janitor-new.css]: overflow-y: auto;

Comment (notriddle) [reply static/css/janitor-new.css]: I *intentionally* put in a permanent scrollbar. It's to keep the centered content from jumping around as it goes from being big enough to scroll to being small enough to not scroll.

Comment (nt1m) [reply static/css/janitor-new.css]: I personally prefer having the content jumping for a tad bit, than having a permanent scrollbar.

**Comment by notriddle** [comment]
Based on ntim's approval, I guess it's suitable.
